google/cloud/storage/retry.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Retry Strategy logic used across google.cloud.storage requests."""
+
+# Should not be in here and only for prototyping
+import six
+import socket
+import requests
+import urllib3
+
+from google.api_core import exceptions
+from google.api_core import retry
```

Comment:
Neat, so this is what we told customers to use as a workaround already right?

Reply:
Also, where are configs around number of attempts or timeouts? Are these values hard-coded into google.api_core.retry?